Dockerfile.QA

```
@@ -384,6 +384,10 @@ RUN rm -fr qa/L0_copyrights qa/L0_build_variants && \
 RUN find qa/pkgs/ -maxdepth 1 -type f -name \
     "tritonserver-*.whl" | xargs -I {} pip3 install --upgrade {}[all]
 
+# Install Triton Frontend Python API
+RUN find qa/pkgs/ -maxdepth 1 -type f -name \
+    "tritonfrontend-*.whl" | xargs -I {} pip3 install --upgrade {}[all]
```

I suppose there shouldn't be more then one file with given pattern.
```suggestion
RUN find qa/pkgs/  -name "tritonfrontend-*.whl" -exec pip3 install --upgrade {}[all]
```